sdks/python/src/opik/api_objects/trace/trace_data.py

```diff
@@ -101,3 +101,31 @@ def _update_attachments(self, attachments: List[Attachment]) -> None:
             self.attachments = attachments
         else:
             self.attachments.extend(attachments)
+
+    @property
+    def trace_start_parameters(self) -> Dict[str, Any]:
```

Comment:
Once we move the logging decision from Opik class to base_track_decorator, we may keep just something like `as_parameters()` method or a similar property that will behave similarly to __dict__ but remove the fields we don't need (e.g. created_by), no need to distinguish between start and end once we no longer need `_trace_start`